docs/discovery/bos/quick-start.md

```diff
@@ -0,0 +1,304 @@
+---
+sidebar_position: 2
+---
+
+
+# Quick Start
+
+To get up and running quickly, take a look at the full-fledged Lido example, which demonstrates 
+how you can interact with smart contracts via ethers.js, style your component, and manage state.
+
+
+#### Fork the component
+
+1. Navigate to [the component](https://bos.gg/#/zavodil.near/widget/Lido)
+2. Click on the menu icon in the top-right corner
+3. Select `Fork`
+4. Feel free to make any changes 
+5. Click on "Save" to deploy the component 
+
+Please note that to deploy the component, you'll need to sign in with a NEAR account and to make a deposit of a small amount of NEAR for the storage cost. This is because BOS uses the NEAR blockchain as its underneath.
+
+
+#### Source code
+
```

Comment:
This is the Lido example? We could put it as a simple example, instead of a quickstart. Since we already have a quickstart.

Reply:
Yes, this is the Lido example that we already have, just with some additional notes in the beginning on how to fork the component.

Reply:
Merged this with our current Lido tutorial